packages/bot/src/transformers.ts

```diff
@@ -380,6 +389,10 @@ export interface Transformers {
       thread: boolean
       type: boolean
       webhookId: boolean
+      call: {
+        participants: boolean
+        endedTimestamp: boolean
```

Comment:
```suggestion
        endedTimestamp: number
```

Reply:
@AwesomeStickz This is also desiredProperties...